qml/Stage/Stage.qml

```diff
@@ -497,6 +497,11 @@ FocusScope {
             PropertyChanges { target: noAppsRunningHint; visible: (root.topLevelSurfaceList.count < 1) }
             PropertyChanges { target: blurLayer; visible: true; blurRadius: 32; brightness: .65; opacity: 1 }
             PropertyChanges { target: wallpaper; visible: false }
+
+            // If we enter spread, remove focus from any app
+            // Focus should not be given back here as that
+            // is handled by spread when selecting app.
+            PropertyChanges { target: topLevelSurfaceList; rootFocus: false }
```

Comment:
It appears that this change is already done with the stage's interactive property: https://github.com/ubports/unity8/blob/5095f7e4b4a762b47aa5c23c6c51398f77977290/qml/Stage/Stage.qml#L103